src/frontend/komponenter/Sak/Personopplysninger.tsx

```diff
@@ -1,12 +1,13 @@
 import * as React from 'react';
 import { IPersonopplysninger } from '../../typer/personopplysninger';
+import AnyData from './AnyData';
 
 export interface IProps {
     data?: IPersonopplysninger;
 }
 
 const Personopplysninger: React.FC<IProps> = ({ data }) => {
-    return <div>{JSON.stringify(data)}</div>;
+    return <AnyData data={data} />;
```

Comment:
Er dette for å vise søknaden til bruker?

Reply:
Søknaden, personopplysninger, overngasstønad, litt annet
Kun temporært, for litt penere visning enn json, vurderte json pretty print også. 

Våre frontend folk kommer nok ikke å begynne med sak før i september, men tenkte det var fint å vise litt av det vi gjør i backend :)